net/nextdns/Portfile

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/nextdns/nextdns v1.11.0
```

Comment:
Can we add `revision` here:
```
revision              0
```

We are wanting to add that to all ports by default.

Reply:
Also, please fix the version prefix here.  It should be:
```
go.setup            github.com/nextdns/nextdns 1.11.0 v
```

and not
```
go.setup            github.com/nextdns/nextdns v1.11.0
```

Reply:
Whoops, I knew that and just forgot. Fixed.